portfolio/src/main/webapp/style.css

```diff
@@ -15,3 +15,23 @@ img{
     width: 40%;
    
 }
+
+div.gallery{
```

Comment:
Add space after 'div.gallery'. Same goes for img, div.gallery:hover, div.gallery img, and div.desc.